.github/workflows/skywalking.yaml

```diff
@@ -583,16 +583,21 @@ jobs:
             config: test/e2e-v2/cases/vm/zabbix/e2e.yaml
           - name: VM Prometheus
             config: test/e2e-v2/cases/vm/prometheus-node-exporter/e2e.yaml
+            env: OTEL_COLLECTOR_VERSION=0.92.0
           - name: VM Telegraf
             config: test/e2e-v2/cases/vm/telegraf/e2e.yaml
           - name: So11y
             config: test/e2e-v2/cases/so11y/e2e.yaml
+            env: OTEL_COLLECTOR_VERSION=0.92.0
           - name: MySQL Prometheus and slowsql
             config: test/e2e-v2/cases/mysql/mysql-slowsql/e2e.yaml
+            env: OTEL_COLLECTOR_VERSION=0.92.0
```

Comment:
This could be on matrix level?

Reply:
> This could be on matrix level?
I will set this as a global env variable with matrix level If we do not need assign values individually for each test.

Reply:
No, we don't need to.